plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/DisabledGlueColumnStatisticsProvider.java

```diff
@@ -57,9 +57,9 @@ public void updateTableColumnStatistics(Table table, Map<String, HiveColumnStati
     }
 
     @Override
-    public void updatePartitionStatistics(Partition partition, Map<String, HiveColumnStatistics> columnStatistics)
+    public void updatePartitionStatistics(Set<StatisticsUpdate> statisticsUpdates)
```

Comment:
you can leave method with old signature as `default` method in the interface, and pass call to new method which takes a `Set`.